src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -328,6 +324,17 @@ private void Page_Loaded(object sender, RoutedEventArgs e)
 			FindName(nameof(InnerNavigationToolbar));
 			FindName(nameof(horizontalMultitaskingControl));
 			FindName(nameof(NavToolbar));
+
+			// Prompt user to review app in the Store
+			if
+			(
+				SystemInformation.Instance.TotalLaunchCount >= 15 &
+				Package.Current.Id.Name == "49306atecsolution.FilesUWP" &&
```

Comment:
Extracting the string value inside a constant?

Reply:
Is there a constant?

Reply:
> Extracting the string value inside a constant?

I think there's no constant, or, case there's one, where is the constant?

Reply:
Looks like this is the only place where we use the package name so we can keep as is.